editor/src/clj/editor/scene_visibility.clj

```diff
@@ -154,20 +154,12 @@
 
   (output outline-name-paths OutlineNamePaths :cached (g/fnk [active-scene] (set (scene-outline-name-paths active-scene))))
 
-  (output outline-name-paths-by-selection-state OutlineNamePathsByBool :cached (g/fnk [outline-name-paths outline-selection]
-                                                                                 (let [selected-outline-name-paths (into [] (keep outline-selection-entry->outline-name-path) outline-selection)
-                                                                                       outline-name-path-below-selection? (fn [outline-name-path]
-                                                                                                                            (boolean (some #(iutil/seq-starts-with? outline-name-path %)
-                                                                                                                                           selected-outline-name-paths)))]
-                                                                                   (iutil/group-into {} #{}
-                                                                                                     outline-name-path-below-selection?
-                                                                                                     outline-name-paths))))
-
-  (output selected-outline-name-paths OutlineNamePaths (g/fnk [outline-name-paths-by-selection-state]
-                                                         (outline-name-paths-by-selection-state true)))
-
-  (output unselected-outline-name-paths OutlineNamePaths (g/fnk [outline-name-paths-by-selection-state]
-                                                           (outline-name-paths-by-selection-state false)))
+  (output selected-outline-name-paths OutlineNamePaths (g/fnk [outline-selection]
+                                                         (set (into [] (keep outline-selection-entry->outline-name-path) outline-selection))))
+
+  (output unselected-outline-name-paths OutlineNamePaths (g/fnk [selected-outline-name-paths outline-name-paths]
+                                                           (set/difference outline-name-paths selected-outline-name-paths)))
+  
```

Comment:
I think we should `:cache` these outputs now that they are no longer trivial to evaluate?